core/systems/mission/patrol.ts

```diff
@@ -88,17 +86,30 @@ export const patrolMissionHandler: MissionHandler = {
         description: transform(template.description),
         rewards: [
           { type: "money", amount: reward },
-          { type: "relation", amount: 1.5, factionId: faction.id },
+          // Add 1 relation point for every 10 minutes of patrol
```

Comment:
sounds a bit op but we'll see once we playtest

Reply:
You may be right 🤔 We'll see later, I'm not sure about money scaling either